docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -426,32 +532,39 @@ Use these commands to start the Sawtooth services:
 
     $ sudo systemctl start sawtooth-rest-api.service
     $ sudo systemctl start sawtooth-poet-validator-registry-tp.service
+    $ sudo systemctl start sawtooth-poet-engine.service
     $ sudo systemctl start sawtooth-validator.service
     $ sudo systemctl start sawtooth-settings-tp.service
     $ sudo systemctl start sawtooth-intkey-tp-python.service
+    $ sudo systemctl start sawtooth-identity-tp.service
 
 You can follow the logs by running:
 
 .. code-block:: console
 
     $ sudo journalctl -f \
     -u sawtooth-validator \
-    -u sawtooth-tp_settings \
+    -u sawtooth-settings-tp \
     -u sawtooth-poet-validator-registry-tp \
+    -u sawtooth-poet-engine \
     -u sawtooth-rest-api \
-    -u sawtooth-intkey-tp-python
+    -u sawtooth-intkey-tp-python \
+    -u sawtooth-identity-tp.service
```

Comment:
Is this a copy/paste error? (.service)

Reply:
I think for consistencies sake we should remove the `.service`. In my limited experience, systemctl and journalctl allow .service to be at the end.

Reply:
@dcmiddle - Thanks for this catch!  The same issue is also in the non-SGX section (plus the additional error of `sawtooth-poet-engine.service`). I'll submit a PR to fix the non-SGX section.